packages/playwright-core/src/utils/registry.ts

```diff
@@ -114,6 +90,8 @@ const DOWNLOAD_PATHS = {
     'mac10.15': 'builds/firefox/%s/firefox-mac-11.zip',
     'mac11': 'builds/firefox/%s/firefox-mac-11.zip',
     'mac11-arm64': 'builds/firefox/%s/firefox-mac-11-arm64.zip',
+    'mac12': 'builds/firefox/%s/firefox-mac-11.zip',
```

Comment:
Just to double check - we reuse 11 build for 12?

Reply:
yes we do